packages/firebase_inappmessaging/ios/firebase_inappmessaging.podspec

```diff
@@ -0,0 +1,24 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+Pod::Spec.new do |s|
+  s.name             = 'firebase_inappmessaging'
+  s.version          = '0.0.1'
+  s.summary          = 'Firebase InAppMessaging Plugin for Firebase'
```

Comment:
This is redundant, maybe "In-app messaging plugin for Firebase"

Reply:
Done.